### DISCUSSION TITLE
Error "ConEmuHk64 git.exe Injecting hooks into PID FAILED" whenever i push, pull, or merge code. Please help me!

### DISCUSSION THREAD

**Discussion by theanh089** [gitextensions/gitextensions#10911] [General]
I always get this error every time I pull, push or merge code. I have tried many different ways but still can't solve it. Please support me. Thank you very much.

![image](https://user-images.githubusercontent.com/54622745/233927538-dd70aea9-f8e3-438d-b83a-ef9af44b9b4e.png)

**Comment by vbjay** [comment]
Verify the conemu files where you have ge installed are unblocked.

**Reply by theanh089** [reply]
I tried unlocking according to the instructions in this link. But they all don't have the security field like in the link:
https://conemu.github.io/en/ZoneId.html

![image](https://user-images.githubusercontent.com/54622745/234175434-695abe8f-5db6-4937-9797-42de40b7ac49.png)

**Comment by vbjay** [comment]
Are you running git under a wsl path?

**Reply by theanh089** [reply]
No!

**Comment by vbjay** [comment]
@Maximus5 thoughts?

**Comment by gerhardol** [comment]
Try disable ConEmu for now.
The core problem should be that the virus program blocks ConEmu.
![image](https://user-images.githubusercontent.com/6248932/234253313-f9516ade-e096-46b1-b7df-f5757ab4b256.png)

**Reply by theanh089** [reply]
i have solved it. thank you

**Reply by theanh089** [reply]
but is there any other way?

**Reply by vbjay** [reply]
Providing details like antivirus being used, the info in help about copy button, does running ge as administrator allow it to work...anything else you can think of, would help.

**Reply by theanh089** [reply]
I don't use any anti-virus software other than Windows 11 Virus & Threat protection. I also tried running GE with administrator privileges and also turning off "Real-time protection". But still not solve my problem. Please support me. Thank!

**Reply by vbjay** [reply]
Click the help ->about menu in GE. Click the copy button next to info. Paste a reply here what it copied.

**Reply by theanh089** [reply]
@vbjay 
- Git Extensions 4.0.2.16100
- Build 25100ec1f7c8da8f798613da74826038af81a50e
- Git 2.40.0.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.16
- DPI 96dpi (no scaling)
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by mstv** [comment]
Can you unblock ConEmu using https://ci.appveyor.com/api/buildjobs/0efq9972bu4tlp80/artifacts/combined.4.1.0.16672-unsigned.zip?
Recently, I got a regarding TaskDialog popup which was able to unblock ConEmu.
